test/node/parallel/test-http-abort.js

```diff
@@ -23,20 +33,24 @@ function test(url) {
   var handle = getHandle(url);
   var req = handle.get(url, function(res) {
     res.on('data', function(chunk) {
-      assert.strictEqual(isAborted, false, 'should not aborted');
+      assert.strictEqual(isAborted, false, `${url} should not aborted`);
       isAborted = true;
       req.abort();
       process.nextTick(function() {
-        assert.strictEqual(eventTriggered, true, 'should trigger event abort');
+        assert.strictEqual(eventTriggered, true, `${url} should trigger event abort`);
       });
       chunks.push(chunk);
     });
 
   });
 
-  req.on('abort', function() {
+  req.on('abort', common.mustCall(function() {
     eventTriggered = true;
-  });
+  }));
+
+  req.socket.on('close', common.mustCall(function() {
+  }));
```

Comment:
Why handle the close on the empty function?

Reply:
to ensure that "close" events can be triggered, that is, the socket can be destroyed